sample-code/src/langchain-orchestration.ts

```diff
@@ -1,39 +1,112 @@
 import { StringOutputParser } from '@langchain/core/output_parsers';
 import { OrchestrationClient } from '@sap-ai-sdk/langchain';
+import {
+  buildAzureContentSafetyFilter,
+  buildLlamaGuardFilter,
+  type OrchestrationModuleConfig
+} from '@sap-ai-sdk/orchestration';
 
 /**
- * ASk GPT about an introduction to SAP Cloud SDK.
+ * Ask GPT about an introduction to SAP Cloud SDK.
  * @returns The answer from ChatGPT.
  */
 export async function invokeChain(): Promise<string> {
-  const orchestrationConfig = {
+  const orchestrationConfig: OrchestrationModuleConfig = {
     // define the language model to be used
     llm: {
-      model_name: 'gpt-35-turbo',
-      model_params: {}
+      model_name: 'gpt-4o'
     },
     // define the template
     templating: {
       template: [
         {
           role: 'user',
-          content: 'Give me a long introduction of {{?input}}'
+          content: 'Tell me about {{?topic}}'
```

Comment:
[pp] I would in general avoid using the term `input` in the input params. It is confusing with the input concept for content filtering.